pom.xml

```diff
@@ -23,7 +23,14 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
-  
+    
+    <dependency>
+      <groupId>junit</groupId>
```

Comment:
Nit: Why did this get moved?

Reply:
The result of a poor conflict resolution, I can put it back.

Reply:
Done